src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/source_filters_table/components/add_filter/__tests__/__snapshots__/add_filter.test.js.snap

```diff
@@ -0,0 +1,40 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AddFilter should render normally 1`] = `
+<EuiFlexGroup
+  alignItems="stretch"
+  component="div"
+  gutterSize="l"
+  justifyContent="flexStart"
+  responsive={true}
+  wrap={false}
+>
+  <EuiFlexItem
+    component="div"
+    grow={10}
+  >
+    <EuiFieldText
+      fullWidth={true}
+      isLoading={false}
+      onChange={[Function]}
+      placeholder="source filter, accepts wildcards (e.g., \`user*\` to filter fields starting with \\\\'user\\\\')"
```

Comment:
Perhaps one too many `\`s 🙂 It shows up in the placeholder:

![screen shot 2018-02-16 at 11 42 05 am](https://user-images.githubusercontent.com/1965714/36326008-7eca1244-130e-11e8-8b9d-4c4a66e3b040.png)